pkg/util/tests/cmd.go

```diff
@@ -0,0 +1,80 @@
+package tests
+
+import (
+	"bufio"
+	"os"
+	"os/exec"
+	"os/user"
+	"strings"
+)
+
+func findK3sExecutable() string {
+	k3sBin := "dist/artifacts/k3s"
+	for {
+		_, err := os.Stat(k3sBin)
+		if err != nil {
+			k3sBin = "../" + k3sBin
+			continue
+		}
+		break
+	}
+	return k3sBin
+}
+
+func IsRoot() bool {
+	currentUser, err := user.Current()
+	if err != nil {
+		return false
+	}
+	return currentUser.Username == "root"
```

Comment:
I'm curious if we should be looking for a uid of 0 rather than the login name as that has the ability to be manipulated.